packages/theme/pages/MyAccount/OrderHistory.vue

```diff
@@ -82,27 +41,29 @@
 import {
   SfTabs,
   SfTable,
-  SfButton,
-  SfProperty,
-  SfLink
+  SfButton
 } from '@storefront-ui/vue';
-import { computed, ref } from '@nuxtjs/composition-api';
+import { computed, useRouter } from '@nuxtjs/composition-api';
 import { useUserOrder, orderGetters } from '@vue-storefront/spree';
 import { AgnosticOrderStatus } from '@vue-storefront/core';
 import { onSSR } from '@vue-storefront/core';
 
 export default {
-  name: 'PersonalDetails',
+  name: 'OrderHistory',
   components: {
     SfTabs,
     SfTable,
-    SfButton,
-    SfProperty,
-    SfLink
+    SfButton
   },
-  setup() {
+  setup(_, context) {
+    const router = useRouter();
     const { orders, search } = useUserOrder();
-    const currentOrder = ref(null);
+
+    const displayOrderDetails = (order) => {
+      const path = `/my-account/order-details/${orderGetters.getId(order).toLowerCase()}`;
```

Comment:
Maybe we should use order's number instead of id in the path? It's customer facing

Reply:
Nevermind, I see it's the order number